CHANGELOG.next.asciidoc

```diff
@@ -223,6 +223,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - Handle unexpectedEOF error in aws-s3 input and enforce retrying using download failed error {pull}42420[42756]
 - Prevent azureblobstorage input from logging key details during blob fetch operations. {pull}43169[43169]
 - Handle special values of accountExpires in the Activedirectory Entity Analytics provider. {pull}43364[43364]
+- Add new grok pattern to system (syslog) integration to capture multiline logs with ISO 8601 timestamps. {pull}43047[43047]
```

Comment:
[Suggestion]
I believe as it's the beats changelog, it's better to reference it as "module" instead of using "integration"